docs/source/tutorial/tutorial_template.md

```diff
@@ -1,24 +1,41 @@
 
-In this section, we discuss the project set-up phase, which is the first part of the [standard development workflow](./spaceflights_tutorial.md#kedro-project-development-workflow). The set-up steps are as follows:
 
-* Create a new project
-* Install dependencies
-* Configure the project
+In this section, we discuss the project set-up phase, which is the first part of the standard development workflow. The setup steps are as follows:
+
+* Create a new project with `kedro new`
+* Install project dependencies with `pip install -r src/requirements.txt`
+* Configure the following in the `conf` folder:
+	* Credentials and any other sensitive information
+	* Logging
 
 ## Create a new project
 
-Navigate to your chosen working directory and run the following to [create a new empty Kedro project](../get_started/new_project.md#create-a-new-project-interactively) using the default interactive prompts:
+If you have not yet set up Kedro, please do so by following the guidelines to [set up a virtual environment with `conda`](../get_started/prerequisites.md#create-a-virtual-environment-with-conda) and [install Kedro](../get_started/install.md).
+
+```{important}
+We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.18.3).
+```
+
+In your terminal window, navigate to the folder you want to store the project and type the following to create an empty project:
 
 ```bash
 kedro new
 ```
 
-When prompted for a project name, enter `Kedro Tutorial`. Subsequently, press enter to accept the default suggestions for `repo_name` and `python_package`. Then navigate to the root directory of the project, `kedro-tutorial`.
+```{note}
+Alternatively, if you want to include a complete set of working example code within the project, generate the project from the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights) by typing `kedro new --starter=spaceflights`.
+```
 
-## Install dependencies
+When prompted for a project name, enter `Kedro Tutorial`. When Kedro has created your project, you can navigate to the [project root directory](./spaceflights_tutorial.md#project-root-directory):
 
-Up to this point, we have not discussed project dependencies, so now is a good time to examine them. We use a `requirements.txt` file to specify a project's dependencies and make it easier for others to run your project. This avoids version conflicts by ensuring that you use same Python packages and versions.
+```bash
+cd kedro-tutorial
+```
+
+## Project dependencies
+
+Kedro projects have a `requirements.txt` file to specify their dependencies and enable sharable projects by ensuring consistency across Python packages and versions.
 
 The generic project template bundles some typical dependencies in `src/requirements.txt`. Here's a typical example, although you may find that the version numbers differ slightly depending on your version of Kedro:
 
```

Comment:
Perhaps we want to make this more readable? We could batch them into commented sections e.g.

```
# code quality packages 
black==22.1.0 # Used for formatting code with `kedro lint`
flake8>=3.7.9, <5.0 # Used for linting code with `kedro lint`
ipython==7.0 # Used for an IPython session with `kedro ipython`
isort~=5.0 # Used for linting code with `kedro lint`
nbstripout~=0.4 # Strips the output of a Jupyter Notebook and writes the outputless version to the original file

# notebook tooling
jupyter~=1.0 # Used to open a Kedro-session in Jupyter Notebook & Lab
jupyterlab~=3.0 # Used to open a Kedro-session in Jupyter Lab

# Pytest + useful extensions
pytest-cov~=3.0 # Produces test coverage reports
pytest-mock>=1.7.1, <2.0 # Wrapper around the mock package for easier use with pytest
pytest~=6.2 # Testing framework for Python code

Reply:
@datajoely Is there a reason that you removed the pinned Kedro dependency in the above?